app/models/fixed_session.rb

```diff
@@ -43,6 +43,7 @@ def after_measurements_created
   end
 
   def update_end_time!
+    # Measurement.time is a local time, so this are both local end times:
     self.end_time = self.measurements.maximum('time')
     self.end_time_local = self.measurements.maximum('time')
     self.last_measurement_at = DateTime.current
```

Comment:
We should change that if we add the utc column to measurement. Or just delete `start_time` and `end_time` from Session.

Reply:
Could you elaborate more on this please?

Reply:
Sure. So we do:
```
self.end_time = self.measurements.maximum('time')
self.end_time_local = self.measurements.maximum('time')
```

And this `maximum('time')` comes from `Measurement.time` column, which as we know is saved as local time. So the `Session.end_time` is the same as `Session.end_local_time` here. 
I think there is no reason to have this data duplicated in the database. So we should:
- either remove the `end_time` column from the db
- or change `end_time` to store the time in utc. (eg by adding the `utc_time` column to Measurements and using it to create `self.end_time = self.measurements.maximum('utc_time')`)

And I think there is a similar problem might be with `start_time` and `start_local_time`.

Reply:
Would you mind creating a ticket somewhere and documenting this please?